content/source/docs/enterprise/api/run.html.md

```diff
@@ -267,4 +267,12 @@ curl \
   --request POST \
   --data @payload.json \
   https://app.terraform.io/api/v2/runs/run-DQGdmrWMX8z9yWQB/actions/discard
-```
\ No newline at end of file
+```
+
+## Available Related Resources
```

Comment:
I think all of these sections should start with a very brief reminder of what this is for (plus a link), right after the header. 

```
The GET endpoints above can optionally return related resources, if requested with [the `include` query parameter](./index.html#inclusion-of-related-resources). The following resource types are available:
```

Unrelatedly, wow the GitHub diff viewer did some violence to this page.